benjaminkott/bootstrap_package
# [FEATURE] Add better language support

Fixes #510 

### Prerequisites

* [x] Changes have been tested on TYPO3 8.7 LTS
* [x] Changes have been tested on TYPO3 9.2-dev

### Description

See issue #510 

### Steps to Validate

1. Assign a language list or `auto` to the constant `page.theme.language.languageValue` with TSCE
2. Complete `config.language.default.*` with TSCE
3. Add a new system languages and complete the existings, disabled will not be shown in auto mode
4. Check language menu at the FE
